### DISCUSSION TITLE
Multivariate fBM?

### DISCUSSION THREAD

**Discussion by hanskersting** [crflynn/stochastic#59] [Ideas]
Hi!

Is there already an implementation of the multidimensional version of the fBM, as e.g. introduced in https://arxiv.org/abs/1007.0828 ?

Best,
Hans

**Comment by crflynn** [comment]
There is another request here for multidimensional implementations in general: https://github.com/crflynn/stochastic/issues/16.
I'd be happy to support the feature if anyone is inclined to contribute but my focus is on other projects at the moment.
